### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by afonsocappelo** [ndeadly/MissionControl#187] [General]
Good afternoon, I'm not sure how to talk to you and I almost gave up ahahah, I'm out of joycon and my salvation would be to use the ipega control, but it doesn't have support for my 9077, could you add? I'm using google translator !

**Comment by ndeadly** [comment]
Hi, I replied to your comment on the other issue. Open a new issue if you believe the controller doesn't work with mission control currently. Otherwise this is a more suitable space if you want to ask questions and receive feedback

**Comment by medwinxyz** [comment]
Hi guys, i tried this and my ipega 9118 didnt detect on my switch. Please add support on it too.
Hw:1.01
PID:35
VID:2
Thanks!

**Reply by ndeadly** [reply]
@medwinxyz please start your own thread in the future rather than hijacking somebody else's. 

Does the controller connect to the Switch at all? I believe you should be using the `A + HOME` button combo to start the controller in Android mode.

 These are not valid VID/PID values, that should be 4-digit hexadecimal numbers (eg. most ipega controllers use `0x1949, 0x0402`) Can you show me the output of my [btdb](https://github.com/ndeadly/MissionControl/releases/download/v0.3.0/btdb.nro) application?

**Reply by medwinxyz** [reply]
Ok.ndeadly. excuse me about this since it was my first time to post.. I will open a new thread and will check the hexavalue.thanks again!
